hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -4067,6 +4081,10 @@ public Collection<String> getOmAdminUsernames() {
     return omAdmins.getAdminUsernames();
   }
 
+  public Collection<String> getOmReadOnlyUsernames() {
```

Comment:
For consistency:

```suggestion
  public Collection<String> getOmReadOnlyAdminUsernames() {
```

(apply change to callers, too)

Reply:
@adoroszlai ok, it's changed now